core/src/main/resources/error/error-classes.json

```diff
@@ -904,6 +904,11 @@
     ],
     "sqlState" : "22003"
   },
+  "INVALID_BOUNDARY" : {
+    "message" : [
+      "Boundary <boundary> is not a valid integer: <value>."
```

Comment:
Precisely, speaking:
- `start` can be 0, Long.MinValue, or [Int.MinValue, Int.MaxValue]
- `stop` - 0, Long.MaxValue, or [Int.MinValue, Int.MaxValue]

Could you, please, add sub-classes like:
```
  "INVALID_BOUNDARY" : {
    "message" : [
      "The boundary <boundary> is invalid: <invalidValue>."
...
    "subClass" : {
      "START" : {
        "message" : [
          "Expected ... ."
        ]
      },
```